cpp/src/aligned.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020-2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <rmm/aligned.hpp>
+
+#include <cassert>
+#include <cstddef>
+#include <cstdint>
+
+namespace RMM_NAMESPACE {
```

Comment:
I believe here we should be using `namespace rmm` since it is the header file which defines our public exported API.